### PR TITLE
Newsletters: add 330 (2024-11-22)

### PR DESCRIPTION
**Pull request by harding**
- [x] Lede, releases/RCs, topic entries @harding 
- [x] Follow up on unwritten news items @harding 
- [x] Client/services section @bitschmidty 
- [x] Notable merges section @Gustavojfe

### CODE REVIEW

**Comment by harding** [comment]
Apologies for opening the PR late this week.

File: _posts/en/newsletters/2024-11-22-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+  about additional specification changes that could allow channels
+  within factories to be [announced][topic channel announcements] but
+  ZmnSCPxj [said][zmnscpxj plug2] he deliberately didn't consider those
+  in his design in order to allow it the specification change to be
+  adopted as fast as possible.
+
+- **Signet activity report:** Anthony Towns [posted][towns signet] to
+  Delving Bitcoin a summary of activity on the default [signet][topic
+  signet] related to proposed soft forks available through [Bitcoin
+  Inquisition][bitcoin inquisition repo].  The post looks at use of
```
Comment (LarryRuane) [inline _posts/en/newsletters/2024-11-22-newsletter.md:41]:
```suggestion
  Inquisition][bitcoin inquisition repo].  The post looks at
```
Or, "the post looks at the use of  [SIGHASH_ANYPREVOUT][topic sighash_anyprevout], including tests ..."

(In other words, remove "usage")

File: _posts/en/newsletters/2024-11-22-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+  [SIGHASH_ANYPREVOUT][topic sighash_anyprevout] usage, including tests
+  of [LN-Symmetry][topic eltoo] and emulation of
+  [OP_CHECKTEMPLATEVERIFY][topic op_checktemplateverify].  It then looks
+  at `OP_CHECKTEMPLATEVERIFY` usage directly, including what are likely
+  several different [vault][topic vaults] constructions and a few data
+  carrier transactions.  Finally, the post looks at usage of
+  [OP_CAT][topic op_cat], including for a proof-of-work faucet (see
```
Comment (LarryRuane) [inline _posts/en/newsletters/2024-11-22-newsletter.md:48]:
```suggestion
  carrier transactions.  Finally, the post looks at
  [OP_CAT][topic op_cat] usage, including for a proof-of-work faucet (see
```

**Comment by Gustavojfe** [comment]
I just added the merge summaries. I removed [LDK #3283][], since it was included in last week's newsletter.

File: _posts/en/newsletters/2024-11-22-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+  [Newsletter #306][news306 powfaucet]), a possible vault or other
+  [covenant][topic covenants], and verification of a [STARK][]
+  zero-knowledge proof.
```
Comment (vostrnad) [inline _posts/en/newsletters/2024-11-22-newsletter.md:51]: Shameless plug: worth mentioning? https://delvingbitcoin.org/t/ctv-apo-cat-activity-on-signet/1257/4

File: _posts/en/newsletters/2024-11-22-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
+  PR also sets P2TR as the default script type for [anchor outputs][topic anchor
+  outputs] spends and for unilateral-close change addresses.
+
+- [Core Lightning #7102][] extends the `generatehsm` command to run
+  non-interactively with command line options. Previously, you could only
+  generate a Hardware Security Module (HSM) secret through an interactive
+  process at the terminal, so this change is particularly useful for automated
+  installations.
+
+- [[Core Lightning #7604][] adds the `bkpr-editdescriptionbypaymentid` and
```
Comment (vostrnad) [inline _posts/en/newsletters/2024-11-22-newsletter.md:157]:
```suggestion
- [Core Lightning #7604][] adds the `bkpr-editdescriptionbypaymentid` and
```

File: _posts/en/newsletters/2024-11-22-newsletter.md [hunk 5]
```diff
@@ -0,0 +1,210 @@
...
+- **Update to LNHANCE proposal:** Moonsettler [posted][moonsettler
+  paircommit delving] to Delving Bitcoin and [also][moonsettler
+  paircommit list] the Bitcoin-Dev mailing list a proposal for a new
+  opcode, `OP_PAIRCOMMIT`, to be added to the LNHANCE soft fork proposal
+  that includes [OP_CHECKTEMPLATEVERIFY][topic op_checktemplateverify]
+  and [OP_CHECKSIGFROMSTACK][topic op_checksigfromstack].  The new
+  opcode allows making a hash commitment to a pair of elements; this is
+  similar to what could be achieved using the proposed [OP_CAT][topic
+  op_cat] concatenation opcode or streaming-SHA opcodes such as those
+  [available][streaming sha] in the Elements [sidechain][topic
```
Comment (bitschmidty) [inline _posts/en/newsletters/2024-11-22-newsletter.md:62]:
```suggestion
  [available][streaming sha] in Elements-based [sidechains][topic
```

File: _posts/en/newsletters/2024-11-22-newsletter.md [hunk 6]
```diff
@@ -0,0 +1,210 @@
...
+  outputs standard, allowing zero-fee transactions with a [dust][topic
+  uneconomical outputs] output to appear in the mempool, provided they are
+  simultaneously spent in a transaction [package][topic package relay]. This
+  change improves the usability of advanced constructs such as connector
+  outputs, keyed and unkeyed ([P2A][topic ephemeral anchors]) anchors, which can
+  benefit the extension of protocols such as LN, [Ark][topic ark], [timeout
+  trees][topic timeout trees], [BitVM2][topic acc], and others. This update
+  builds on existing features such as 1P1C relays, [TRUC][topic v3 transaction
+  relay] transactions, and [sibling eviction][topic kindred rbf]. See Newsletter
+  [#328][news328 ephemeral].
```
Comment (bitschmidty) [inline _posts/en/newsletters/2024-11-22-newsletter.md:144]:
```suggestion
  relay] transactions, and [sibling eviction][topic kindred rbf] (see Newsletter
  [#328][news328 ephemeral]).
```

File: _posts/en/newsletters/2024-11-22-newsletter.md [hunk 7]
```diff
@@ -0,0 +1,210 @@
...
+
+- [Core Lightning #7833][] enables the [offers][topic offers] protocol by
+  default, removing its previous experimental status. This follows the merging
+  of its PR into the BOLTs repository (See Newsletter [#323][news323 offers]).
```
Comment (bitschmidty) [inline _posts/en/newsletters/2024-11-22-newsletter.md:148]:
```suggestion
  of its PR into the BOLTs repository (see Newsletter [#323][news323 offers]).
```

File: _posts/en/newsletters/2024-11-22-newsletter.md [hunk 8]
```diff
@@ -0,0 +1,210 @@
...
+
+- [Core Lightning #7799][] introduces the `xpay` plugin to send payments by
+  constructing optimal [multipath payments][topic multipath payments], using the
+  `askrene` plugin (See Newsletter [#316][news316 askrene]) and the
```
Comment (bitschmidty) [inline _posts/en/newsletters/2024-11-22-newsletter.md:152]:
```suggestion
  `askrene` plugin (see Newsletter [#316][news316 askrene]) and the
```

**Comment by harding** [comment]
Made edits for all feedback (thanks @LarryRuane @vostrnad @bitschmidty !), added lede, removed releases/RCs section (none found), reviewed contributions from @bitschmidty and @Gustavojfe (extra great job this week with the deep linking!), and added topic entries.  I also took steps to follow up on the two news items I didn't get to this week.

Thanks everyone!

File: _posts/en/newsletters/2024-11-22-newsletter.md [hunk 9]
```diff
@@ -0,0 +1,213 @@
...
+- **Spark layer two protocol announced:**
+  [Spark][spark website] is an offchain, [statechain][topic statechains]-like
+  protocol that supports the Lightning Network.
+
+- **Unify wallet announced:**
+  [Unify][unify github] is a [BIP78][]-compatible [payjoin][topic payjoin]
+  wallet that uses Bitcoin Core and coordinates [PSBTs][topic psbt] over nostr.
+
+- **bitcoinutils.dev launches:**
+  The [bitcoinutils.dev][] website provides a variety of Bitcoin utilites
```
Comment (azuchi) [inline _posts/en/newsletters/2024-11-22-newsletter.md:108]:
```suggestion
  The [bitcoinutils.dev][] website provides a variety of Bitcoin utilities
```

File: _posts/en/newsletters/2024-11-22-newsletter.md [hunk 10]
```diff
@@ -0,0 +1,213 @@
...
+  at `OP_CHECKTEMPLATEVERIFY` usage directly, including what are likely
+  several different [vault][topic vaults] constructions and a few data
+  carrier transactions.  Finally, the post looks at
+  [OP_CAT][topic op_cat] usage, including for a proof-of-work faucet (see
+  [Newsletter #306][news306 powfaucet]), a possible vault or other
+  [covenant][topic covenants], and verification of a [STARK][]
+  zero-knowledge proof.
+
+  Vojtěch Strnad [replied][strnad i.o] that he was inspired by Towns's
+  post to create a [block explorer][topic block explorers] that lists
```
Comment (vostrnad) [inline _posts/en/newsletters/2024-11-22-newsletter.md:61]: Is it really a "block explorer" when it doesn't actually allow exploring blocks and only shows a subset of transactions? I'd slightly prefer just "explorer" or even "website":
```suggestion
  post to create a website that lists
```

Comment (harding) [reply _posts/en/newsletters/2024-11-22-newsletter.md]: Updated.

File: _posts/en/newsletters/2024-11-22-newsletter.md (hunk 9 above)
Comment (harding) [reply _posts/en/newsletters/2024-11-22-newsletter.md]: Thanks!

**Review by bitschmidty — COMMENTED** [review]
ACK lede, topics with couple changes: Reworded some topics to include the block explorer->website wording

**Pull request merged**